app/controllers/concerns/curation_concerns/curation_concern_controller.rb

```diff
@@ -203,5 +204,20 @@ def search_result_document(search_params)
         raise CanCan::AccessDenied.new(nil, :show) if document_list.empty?
         document_list.first
       end
+
+      def authorize_workflow_role!
```

Comment:
Doesn't CanCan raise `CanCan::AccessDenied`? Not sure how to get CanCan to raise `WorkflowException`.